src/main/resources/config.yml

```diff
@@ -167,6 +167,7 @@ clan:
         - treasurer:
             display-name: "&6Treasurer"
             permissions: [ bank.balance, bank.deposit, bank.withdraw ]
+    default_rank: "recruit"

```

Comment:
We use - instead of _ in config.yml (the previous comment was about the metrics thing)